src/gcp_scanner/scanner.py

```diff
@@ -184,246 +212,40 @@ def crawl_loop(initial_sa_tuples: List[Tuple[str, Credentials, List[str]]],
         logging.error('Try removing the %s file and restart the scanner.',
                       output_file_name)
 
-      if is_set(scan_config, 'iam_policy'):
-        # Get IAM policy
-        project_result['iam_policy'] = CrawlerFactory.create_crawler(
-          'iam_policy',
-        ).crawl(
-          project_id,
-          ClientFactory.get_client('cloudresourcemanager').get_service(
+      for crawler_name, client_name in crawl_client_map.items():
+        if is_set(scan_config, crawler_name):
+          crawler_config = {}
+          if scan_config is not None:
+            crawler_config = scan_config.get(crawler_name)
+          # add gcs output path to the config.
+          # this path is used by the storage bucket crawler as of now.
+          crawler_config['gcs_output_path'] = gcs_output_path
```

Comment:
I wonder if there is a better solution rather than sending it into every crawler. We can think about that in the future but for now it is fine.

Reply:
Right, this would be a good refinement ticket. Will create once once this PR is complete.